packages/next/build/swc/npm/linux-x64-gnu/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "next-swc-linux-x64-gnu",
+  "name": "@next/swc-linux-x64-gnu",
```

Comment:
Did we initialize these packages on `npm` already with 0.0.0?

Reply:
No ,`0.0.0` is more of a placeholder as the versions get overwritten in the `publish-native.js` script. The updated versions are not being committed at the moment because it seemed kind of weird to commit it after the tag, but let me know if you have other ideas.